GVFS/GVFS.Service/GvfsService.cs

```diff
@@ -386,85 +386,6 @@ private void CreateAndConfigureUpgradeLogDirectory()
             string upgradeLogsPath = ProductUpgraderInfo.GetLogDirectoryPath();
             DirectorySecurity upgradeLogsSecurity = this.GetUpgradeLogsDirectorySecurity(upgradeLogsPath);
             Directory.CreateDirectory(upgradeLogsPath, upgradeLogsSecurity);

```

Comment:
It looks like there is still a problem in this scenario:

1. Upgrade to a version with these changes
2. GVFS.Service never starts
3. User runs `gvfs upgrade` from an elevated CMD prompt
4. The new logs directory will not have the correct ACLs, and it will prevent non-elevated users from being able to create upgrade log files.

Reply:
Addressed this by making sure the logs directory is created with the correct ACLs when running from an elevated CMD prompt